src/routes/user/locale.ts

```diff
@@ -1,46 +0,0 @@
-import { gqlSdk } from '@/utils/gqlSDK';
```

Comment:
Why was this file removed?

Reply:
This handler was not activated anywhere (not linked to an express route)